docs/running.md

```diff
@@ -43,13 +43,25 @@ replica and be unconcerned, as we still have two copies of the data
 left for redundancy. This is configurable: you can run with more
 redundancy or less, depending on your risk appetite.
 
-### Index schema
+### Schema
+
+#### Schema periodic table
+
+The periodic table from argument (`--dynamodb.periodic-table.from=<date>` if
+using command line flags, the `from` field for the first schema entry if using
+YAML) should be set to the date the oldest metrics you will be sending to
+Cortex. Generally that means set it to the date you are first deploying this
+instance. If you use an example date from years ago table-manager will create
+hundreds of tables. You can also avoid creating too many tables by setting a
+reasonable retention in the table-manager.
```

Comment:
could mention the full name here `-table-manager.retention-period` or link to its description in arguments.md